hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -423,8 +424,9 @@ private long calculateMaxSteps(BalancerClusterState cluster) {
             maxSteps);
       }
     }
-    LOG.info("start StochasticLoadBalancer.balancer, initCost=" + currentCost + ", functionCost="
-        + functionCost() + " computedMaxSteps: " + computedMaxSteps);
+    LOG.info("Start StochasticLoadBalancer.balancer, initial weighted average imbalance={}, "
```

Comment:
imbalance?

(Looked it up... "lack of proportion or relation between corresponding things." <= Nice)

Reply:
Could we say anything here on what these values mean ? For the operator? (Need a ',' after the functionCost).

Reply:
the functionCost has a ";" at the end so I removed the ",". The function cost explain the values. Here is the sample log from runs again which I posted above: functionCost=RegionCountSkewCostFunction : (multiplier=500.0, imbalance=0.0, balanced); PrimaryRegionCountSkewCostFunction : (not needed); MoveCostFunction : (multiplier=7.0, imbalance=0.0, balanced); RackLocalityCostFunction : (multiplier=15.0, imbalance=0.0, balanced); TableSkewCostFunction : (multiplier=35.0, imbalance=1.0); RegionReplicaHostCostFunction : (not needed); RegionReplicaRackCostFunction : (not needed); ReadRequestCostFunction : (multiplier=5.0, imbalance=0.0, balanced); CPRequestCostFunction : (multiplier=5.0, imbalance=0.0, balanced); WriteRequestCostFunction : (multiplier=5.0, imbalance=0.0, balanced); MemStoreSizeCostFunction : (multiplier=5.0, imbalance=0.0, balanced); StoreFileCostFunction : (multiplier=5.0, imbalance=0.0, balanced);